include/macros.h

```diff
@@ -27,9 +27,19 @@
 #define YEARS_CHILD 5
 #define YEARS_ADULT 17
 #define LINK_IS_CHILD (gSaveContext.linkAge != 0)
+#define LINK_IS_NOT_ADULT (gSaveContext.linkAge == 1)
+
 #define LINK_IS_ADULT (gSaveContext.linkAge == 0)
+#define LINK_IS_NOT_CHILD (gSaveContext.linkAge != 1)
+
 #define LINK_AGE_IN_YEARS (LINK_IS_CHILD ? YEARS_CHILD : YEARS_ADULT)
 
+#define IS_NIGHT (gSaveContext.nightFlag != 0)
+#define IS_NOT_DAY (gSaveContext.nightFlag == 1)
+
+#define IS_DAY (gSaveContext.nightFlag == 0)
+#define IS_NOT_NIGHT (gSaveContext.nightFlag != 1)
+
```

Comment:
this is relevant regarding these macro changes https://github.com/zeldaret/oot/pull/545#discussion_r549807437

In general, I think we should be careful to talk about these kinds of changes in the discord before implementing them, so we dont end up in a situation where you have to revert work that you spent lots of time on